src/components/Job.jsx

```diff
@@ -38,8 +38,7 @@ const Job = () => {
         .get(`${server}/hypercube/`, {
           params: {
             hypercube_token: token.substring(3)
-          },
-          headers: { "X-Fields": fields.join(", ") }
+          }
```

Comment:
Was there an issue here?

Reply:
Yeah, I realized it today. get `/Hypercube` did not have `X-Fields` and when I implemented that, it actually breaks the functionality for displaying hypercube jobs because those fields do not include `finished`, etc